auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The gRPC Authors
+ * Copyright 2016,2022 The gRPC Authors
```

Comment:
This is fine, but we don't update the date because it really doesn't matter (since copyright is so long). This file has definitely changed since 2016.

Reply:
Reverted